src/main/java/by/skopinau/transactional/outbox/demo/service/OrderService.java

```diff
@@ -0,0 +1,7 @@
+package by.skopinau.transactional.outbox.demo.service;
+
+import java.util.Map;
+
+public interface OrderService {
+    Map<String, Object> createOrder(Map<String, Object> orderMap);
+}
```

Comment:
Remove this interface because you don't use RPC (remote procedure call). For simple microservice, no need to create excess interfaces.

Reply:
done